src/main/resources/application.yaml

```diff
@@ -98,7 +98,7 @@ feign:
     headers:
       - authorization
       - serviceauthorization
-userProfUrl: ${USER_PROFILE_URL:http://rd-user-profile-api-aat.service.core-compute-aat.internal}
+userProfUrl: ${USER_PROFILE_URL:http://rd-user-profile-api-pr-575.service.core-compute-preview.internal}
```

Comment:
any reason the default value being changed from aat ?

Reply:
Currently, the UP has not whitelisted the CW Ref api. Hence used the PR 575. Before going into higher env, we should change it to aat.

Reply:
I have put a comment in RDCC-2059 for this pending piece of work.

Reply:
thanks :)